packages/utilities/src/AutoScroll.ts

```diff
@@ -26,21 +27,22 @@ export class AutoScroll {
   private _isVerticalScroll!: boolean;
   private _timeoutId?: number;
 
-  constructor(element: HTMLElement) {
+  constructor(element: HTMLElement, win?: Window) {
+    const theWin = win ?? getWindow(element)!;
     this._events = new EventGroup(this);
     this._scrollableParent = findScrollableParent(element) as HTMLElement;
 
     this._incrementScroll = this._incrementScroll.bind(this);
-    this._scrollRect = getRect(this._scrollableParent);
+    this._scrollRect = getRect(this._scrollableParent, win);
```

Comment:
You probably should be using `theWin` here

```suggestion
    this._scrollRect = getRect(this._scrollableParent, theWin);
```